Functions/It.ps1

```diff
@@ -79,14 +79,14 @@ function Add-Numbers($a, $b) {
 }
 
 Describe "Add-Numbers" {
-    $testCases = @(
+    $TestCases = @(
```

Comment:
Please revert this, and the usages. `$testCases` is a "local" variable, and those should be lowercase.